shared/src/main/scala/amf/core/internal/plugins/syntax/SyamlSyntaxParsePlugin.scala

```diff
@@ -18,8 +55,8 @@ object SyamlSyntaxParsePlugin extends AMFSyntaxParsePlugin with PlatformSecrets
     if (text.length() == 0) SyamlParsedDocument(YDocument(YNode.Null))
     else {
       val parser = getFormat(mediaType) match {
-        case "json" => JsonParserFactory.fromCharsWithSource(text, ctx.rootContextDocument)(ctx.eh)
-        case _      => YamlParser(text, ctx.rootContextDocument)(ctx.eh).withIncludeTag("!include")
+        case "json" => JsonParserFactory.fromCharsWithSource(text, ctx.rootContextDocument)(new SyamlAMFErrorHandler(ctx.eh))
```

Comment:
errorhandler can be extracted to variable. Could even be an private method of the plugin